gitapp/lib/user_details/user_repos_list_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:gitapp/constants/constants.dart';
+
+Column userReposListWidget(UserRepo) {
```

Comment:
Same as above. Naming convention.

Reply:
Also please create your own subclass for this widget

Reply:
Solved